portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,54 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Kayla Leung Portfolio</title>
+    <title>Kayla Leung</title>
+
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/b99e675b6e.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
   </head>
   <body>
-    <div id="content">
-      <h1>Kayla Leung</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/kaylaleung.jpeg"/>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <!-- Left Side Bar -->
+  <div class="wrapper">
+    <div class="sidebar">
+
+        <div class="avatar">
+          <img src="images/kaylaleung.jpeg">
+        </div>
+
+        <div class="role">
+          <h2>Kayla Leung</h2>
+          <h4>Software Engineering Intern</h4>
+        </div>
+
+        <div class="social_media">
+          <ul>
+            <li><a href="https://www.facebook.com/kaylayleung" target="_blank"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="https://www.instagram.com/kaylaleung/" target="_blank"><i class="fab fa-instagram"></i></a></li>
```

Comment:
Up to you, but consider using protocol-relative URLs: //www.instagram.com/kaylaleung/

Reply:
Change doesn't work yet since I'm testing locally. Will make URL changes when deploying it to a live server.